website/templates/util/render_node.mako

```diff
@@ -127,23 +160,31 @@
             % if not summary['primary'] and 'write' in user['permissions'] and not node['is_registration']:
                 ## Allow deletion of pointers, even if user doesn't know what they are deleting
                 <span class="pull-right">
-                    <i class="fa fa-times remove-pointer pointer" data-id="${summary['id']}" data-toggle="tooltip" title="Remove link"></i>
+                    <i class="fa fa-times remove-pointer pointer" data-id="${summary['id']}"
+                    data-toggle="tooltip" title="Remove link"></i>
                 </span>
             % endif
         </p>
     </li>
-
 % endif
 </div>
 <script type="text/javascript">
```

Comment:
Are there other places where js is being embedded like this? Might consider moving to its own file.